v3/src/components/case-table/case-table-tool-shelf-button.tsx

```diff
@@ -91,15 +91,20 @@ export const CaseTableToolShelfMenuList = observer(function CaseTableToolShelfMe
           const tileTitle = caseMetadata?.title ? caseMetadata?.title
                                                 : tableTile?.title ? tableTile?.title : dataset.dataSet.name
           return (
-            <MenuItem key={`${dataset.dataSet.id}`} onClick={()=>handleOpenDataSetTable(dataset)}>
+            <MenuItem key={`${dataset.dataSet.id}`} onClick={()=>handleOpenDataSetTable(dataset)}
+              data-testid={`toolshelf-table-${tileTitle}`}>
               {tileTitle}
-              <TrashIcon className="tool-shelf-menu-trash-icon"
+              <TrashIcon className="toolshelf-menu-trash-icon"
```

Comment:
I would rather we standardize on the hyphenated form, which doesn't trigger warnings from spell-check extensions, for instance.

Reply:
I've pushed a commit for this.